module/helpers/utils.mjs

```diff
@@ -445,3 +445,11 @@ export function itemIsIdentical(a, b) {
 
     return compendiumSource && name & description;
 }
+
+export function waitForDiceSoNice(message, update) {
```

Comment:
Is there a reason you can't make this an async function you can await on before doing the update, that doesn't involve passing the update here?